BepInEx.PatcherProvider/BepInExPatcherLoadContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+
+namespace BepInEx.PatcherProvider;
+
+public class BepInExPatcherLoadContext : IPluginLoadContext
```

Comment:
These shouldn't be public because people will reference `BepInEx/core/*.dll`, *even when a nuget package exists and is better...*

Reply:
Changed to internal